packages/rollup-plugin-html/README.md

```diff
@@ -452,19 +505,31 @@ All configuration options are optional if an option is not set the plugin will f
 
 Type: `string`
 
-Name of the generated HTML file. If `inputPath` is set, defaults to the `inputPath` filename, otherwise defaults to `index.html`.
+Name of the generated HTML file. If `files` is set, defaults to the `files` filename, otherwise defaults to `index.html`.
 
-### inputPath
+### files
 
-Type: `string`
+Type: `string|strings[]`
 
-Path to the HTML file to use as input. Modules in this file are bundled and the HTML is used as the template for the generated HTML file.
+Paths to the HTML file to use as input. Modules in this files are bundled and the HTML is used as the template for the generated HTML file. Does support glob patterns.
 
-### inputHtml
+### rootDir
 
 Type: `string`
 
-Same as `inputPath`, but provides the HTML as a string directly.
+Path where the `files` should start look.
```

Comment:
`rootDir` used to be only for HTML files which are provided as strings, in order to indicate relative to where relative imports should be resolved. From what I can tell it's now extended to also be where to start globbing from, and which part to strip when not flattening the filename. We should explain that here.